### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by francksefu**
Set up React Testing Library. -> done
Write unit tests for the files operate.js and calculate.js using Jest. -> done
Create unit tests for all React components: -> done
Use Jest snapshots to test the components. -> done
Use React Testing Library to simulate user interaction. -> done
Run your full tests suite using Jest. All tests should pass. -> done

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+import Quotes from './components/Quotes';
 
+// import TestRenderer from 'react-test-renderer';
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  expect(render(<Calculator />)).toMatchSnapshot();
+  expect(render(<Home />)).toMatchSnapshot();
+  expect(render(<Quotes />)).toMatchSnapshot();
```
Comment (Shisui6) [inline src/App.test.js:11]: Nice job using jest snapshots to test your components✨. However, according to the project requirements, you are also supposed to write unit tests for **all** your components using the react testing library. The react testing library provides many useful functions for interacting with your components during testing. Take a look at this [tutorial](https://testing-library.com/docs/react-testing-library/example-intro) to see examples of the library in action. You can also have a look at this [cheatsheet](https://testing-library.com/docs/react-testing-library/cheatsheet) to see common queries used in the library. 

Please create separate test files for each of your components using functions from the react testing library🙏

File: src/App.test.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render } from '@testing-library/react';
+import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
+// import TestRenderer from 'react-test-renderer';
 test('renders learn react link', () => {
```
Comment (Shisui6) [inline src/App.test.js:8]: [Optional] It's considered best practice to organize all your test files in a single `__tests__` folder in your `src` directory. This way, your test files are easy to locate and categorize. I strongly suggest you implement this practise👍

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @francksefu, @ghreza-crypto, and @Ali-0111

# Requires Changes 🍀
You have done a great job so far!👏👏
Your project is almost there. We just need to work on a few issues🪲.

## Highlights
- Tests are passing ✅
- Added tests for logic files ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

_Check the comments under the review._

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
## Hi @francksefu 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**